homeassistant/components/qnap/sensor.py

```diff
@@ -407,10 +406,10 @@ def native_value(self):
 
         data = self.coordinator.data["bandwidth"][self.monitor_device]
         if self.entity_description.key == "network_tx":
-            return round_nicely(data["tx"] / 1024 / 1024)
+            return data["tx"] / 1024 / 1024
```

Comment:
We should remove the calculations as well, and use a correct native unit of measurement and a suggest unit of measurement and let Home Assistant handle the conversions.

Reply:
Nice, done.